src/Tests/Microsoft.NET.Sdk.Web.Tests/PublishTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Text.Json.Nodes;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Sdk.Web.Tests
+{
+    public class PublishTests : SdkTest
+    {
+        public PublishTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [RequiresMSBuildVersionTheory("17.0.0.32901")]
```

Comment:
Why is this particular MSBuild version needed?

Reply:
I don't believe it is needed, i copy/paste from a different test and might forgot to clean up. Let me check and fix it.